docs/guides/advanced/alternative-clients.md

```diff
@@ -44,3 +44,70 @@ exporter = OTLPSpanExporter(
     headers={'Authorization': 'your-write-token'},
 )
 ```
+
+
+## Example with Rust
+
+First, set up a new Cargo project:
+
+```sh
+cargo new --bin otel-example && cd otel-example
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://logfire-api.pydantic.dev
+export OTEL_EXPORTER_OTLP_HEADERS='Authorization=your-write-token'
+```
+
+Update the `Cargo.toml` and `main.rs` files with the following contents:
+
+```toml
+# Cargo.toml
+[package]
+name = "otel-example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+opentelemetry = { version = "0.24", default-features = false, features = ["trace"] }
```

Comment:
Does the version need to be specified? The current latest is 0.26. Same for the below.

Reply:
Updated to use `"*"`. I think unless we check this in CI it bitrots, but hey, we can live with that.